spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/ldap/embedded/EmbeddedLdapAutoConfiguration.java

```diff
@@ -107,6 +109,28 @@ public InMemoryDirectoryServer directoryServer() throws LDAPException {
 					this.embeddedProperties.getCredential().getUsername(),
 					this.embeddedProperties.getCredential().getPassword());
 		}
+
+		Schema schema = null;
+		if (this.embeddedProperties.getValidation().isEnabled()) {
+			schema = Schema.getDefaultStandardSchema();
+			String schemaLocation = this.embeddedProperties.getValidation().getSchema();
+			if (StringUtils.hasText(schemaLocation)) {
+				try {
+					Resource resource = this.applicationContext
+							.getResource(schemaLocation);
+					if (resource.exists()) {
+						schema = Schema.mergeSchemas(schema,
+								Schema.getSchema(resource.getFile()));
+					}
+				}
+				catch (Exception ex) {
+					throw new IllegalStateException(
+							"Unable to load schema " + schemaLocation, ex);
+				}
+			}
+		}
+		config.setSchema(schema);
```

Comment:
Yes, setting the schema to `null` is what disables the validation. Should it be more explicit with something like https://github.com/jgribonvald/embedded-ldap/blob/master/src/test/java/org/esco/test/ldap/LdapServerRule.java#L69-L76?